documentation/asciidoc/computers/configuration/device-tree.adoc

```diff
@@ -991,3 +991,121 @@ Further ports can be controlled by the DT, for more details see <<part3,section
+
+`boot-mode` - 32-bit integer
+
+The boot-mode used to load the kernel. See xref:raspberry-pi.adoc#BOOT_ORDER[BOOT_ORDER].
+
+`overlay_prefix` - string
+
+The xref:config_txt.adoc#overlay_prefix[overlay_prefix] string selected by `config.txt`.
+
+`os_prefix` - string
+
+The xref:config_txt.adoc#os_prefix[os_prefix] string selected by `config.txt`.
+
+`partition` - 32-bit integer
+
+The partition number used during boot.
+
+`pm_rsts` - 32-bit integer
+
+The value of the `PM_RSTS` register during boot.
+
+`rpi-boardrev-ext` - 32-bit integer
+
+The extended board revision code from xref:raspberry-pi.adoc#otp-register-and-bit-definitions[OTP row 33].
+
+`rpi-country-code` - 32-bit integer
+
+The country code used used by https://github.com/raspberrypi-ui/piwiz[PiWiz] - Pi400 only.
+
+==== BCM2711 bootloader properties `/chosen/bootloader`
+The following properties are specific to BCM2711 SPI EEPROM bootloader.
+
+`build_timestamp` - 32-bit integer
+
+The UTC build time for the EEPROM bootloader.
+
+`capabilities` - 32-bit integer
+
+This bit field which describes the features supported by the current bootloader. This may be used to check whether a feature (e.g. USB boot) is supported before enabling it in the bootloader EEPROM config.
+
+|===
+| Bit | Feature
+
+| 0
+| xref:raspberry-pi.adoc#usb-mass-storage-boot[USB boot] using the VLI USB host controller.
+
+| 1
+| xref:remote-access.adoc#network-boot-your-raspberry-pi[Network boot]
+
+| 2
+| xref:raspberry-pi.adoc#fail-safe-os-updates-tryboot[TRYBOOT]
+
+| 3
+| xref:raspberry-pi.adoc#usb-mass-storage-boot[USB boot] using the BCM2711 USB host controller.
+
+| 4
+| xref:raspberry-pi.adoc#boot_ramdisk[RAM disk - boot.img]
+
+| 5
+| xref:raspberry-pi.adoc#nvme-ssd-boot[NVMe boot]
+
+| 6
+| https://github.com/raspberrypi/usbboot/blob/master/Readme.md#secure-boot[Secure Boot]
+|===
+
+`update_timestamp` - 32-bit integer
+
+The UTC update timestamp set by `rpi-eeprom-update`.
+
+`signed_boot` - 32-bit integer
+
+If secure-boot is enabled then this bit field will be non-zero. The individual bits indicate the current secure-boot configuration.
+
+|===
+| Bit | Description
+
+| 0
+| `SIGNED_BOOT` was defined in the EEPROM config file.
+
+| 1
+| Reserved.
+
+| 2
+| The ROM development key has been revoked.  See xref:raspberry-pi.adoc#revoke_devkey[revoke_devkey].
+
+| 3
+| The customer public key digest has been written to OTP. See xref:raspberry-pi.adoc#program_pubkey[program_pubkey].
+
+| 4..31
+| Reserved.
+|===
+
+`version` - string
+
+The Git version string for the bootloader.
+
+==== BCM2711 USB boot properties `/chosen/bootloader/usb`
+The following properties are defined if the system was booted from USB. These may be used by the USB to uniquely identify the USB boot device.
+
+`usb-version` - 32-bit integer
+
+The USB major protocol version (2 or 3).
+
+`route-string` - 32-bit integer
+The USB route-string identifier for the device.
```

Comment:
Does this need additional explanation, or will anyone who needs this info already know what this means? :shrug:

Reply:
I'll append "as defined by the USB 3 specification"